README.md

```diff
@@ -8,7 +8,20 @@ Tooling and infrastructure to enable cloud nowcasting.
 
 ## Installation
 
-From source (development mode):
+### For users:
+
+```zsh
+git clone https://github.com/alan-turing-institute/cloudcasting
+cd cloudcasting
+python -m pip install .
+```
+
+To run metrics on GPU:
+
+```zsh
+python -m install --upgrade "jax[cuda12]"
```

Comment:
Does this mean by default the calculations will be run on the GPU if available?

Reply:
Yes, `jax.default_backend()` should default to `'gpu'`. The strict check would be to change messaging when running validation if this returned `'cpu'`.

Reply:
A sharp edge here is if we happen to have tensorflow installed in our HPC env, where it will take priority access of GPUs. The workaround is 
```python
import tensorflow as tf
tf.config.experimental.set_visible_devices([], "GPU")
```
but we may not run into this!